sdk/tests/conformance/reading/read-pixels-test.html

```diff
@@ -124,80 +126,147 @@
     }
   }
 
-  var badFormats = [
+  continueTestPart2();
+}
+
+function continueTestPart2() {
+  var invalidFormat = [gl.DEPTH_COMPONENT, gl.DEPTH_STENCIL, desktopGL.R8, gl.RGBA4];
+  if (contextVersion < 2) {
+  // They are valid in WebGL 2 or higher
+    invalidFormat = invalidFormat.concat([gl.LUMINANCE, gl.LUMINANCE_ALPHA, desktopGL.RED, desktopGL.RG_INTEGER, desktopGL.RGBA_INTEGER]);
+  }
+
+  var invalidTypeInfo = [
+    {type: gl.UNSIGNED_SHORT,                 dest: new Uint16Array(4)},
+    {type: gl.SHORT,                          dest: new Int16Array(4)},
+    {type: desktopGL.UNSIGNED_INT_24_8,       dest: new Uint32Array(4)}
+  ];
```

Comment:
I think this is incorrect, as these types still can be the implementation selected type.